aerleon/lib/openconfig.py

```diff
@@ -148,6 +163,17 @@ def ConvertToDict(
             protos = ['none']
 
         ace_dict = copy.deepcopy(term_dict)
+
+        # options
+        if self.term.option:
+            for opt in [str(x) for x in self.term.option]:
```

Comment:
Are there cases when options are not a string?

Reply:
Indeed, looks like not. I copied the logic from the Juniper's generator.